<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="22611" started-at="2020-05-07T11:26:23 IST" finished-at="2020-05-07T11:26:46 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="22611" started-at="2020-05-07T11:26:23 IST" finished-at="2020-05-07T11:26:46 IST">
      <class name="com.Testcases.TestBMW">
        <test-method status="PASS" signature="startUp()[pri:0, instance:com.Testcases.TestBMW@69a3d1d]" name="startUp" is-config="true" duration-ms="13349" started-at="2020-05-07T11:26:23 IST" finished-at="2020-05-07T11:26:36 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startUp -->
        <test-method status="FAIL" signature="Model()[pri:0, instance:com.Testcases.TestBMW@69a3d1d]" name="Model" duration-ms="9177" started-at="2020-05-07T11:26:37 IST" finished-at="2020-05-07T11:26:46 IST">
          <exception class="java.lang.NoClassDefFoundError">
            <message>
              <![CDATA[org/apache/xmlbeans/XmlObject]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/xmlbeans/XmlObject
at com.pages.Bmwclass.condition(Bmwclass.java:76)
at com.Testcases.TestBMW.Model(TestBMW.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: org.apache.xmlbeans.XmlObject
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 30 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NoClassDefFoundError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Model -->
      </class> <!-- com.Testcases.TestBMW -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
